catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -203,8 +342,7 @@
                             guava,
                             notifications,
                             activities,
```

Comment:
It is important.  Right now the hazelcast test jar is going to be embedded as well since it has the same artifact id.